lib/db/database_dumper.rb

```diff
@@ -2,168 +2,146 @@
 #  initial plan is to not provide clear option
 #
 require File.join(Rails.root, 'lib', 'tasks', 'rake_task_helpers', 'dumper_utils')
+require File.join(Rails.root, 'lib', 'db', 'database_anonymizer_tasks')
+
 
 class DatabaseDumper
 
   attr_reader :outcome_files, :bucket_key_id, :bucket_access_key, :bucket
 
+  MAX_NUM_OF_RECORDS_PER_GROUP = 10000
   TABLES_TO_BE_EXCLUDED = ["reports", "search_queries", "sessions", "versions"]
+  CLASSES_TO_ANONYMISE = [Team, TeamProperty, ::Warehouse::CaseReport, Case::Base, User, CaseTransition, CaseAttachment]
 
-  def initialize(anonymize, tag, is_store_to_s3_bucket, s3_bucket: nil)
-    @anonymize = anonymize
-    @anonymizer = nil
-    @db_connection_url = ENV['DATABASE_URL'] || 'postgres://localhost/correspondence_platform_development'
-    @s3_bucket = s3_bucket
-    @outcome_files = []
+
+  def initialize(tag, running_mode = 'tasks', is_store_to_s3_bucket = true, s3_bucket_setting = nil)
+    ####
+    # tag:  part of the file name for  the anonymised data, it is ID of the anonymised db copy on s3 bucket
```

Comment:
We should probably add this to a "how to" document of some sort.

Reply:
Will add a follow-up ticket